README.rst

```diff
@@ -22,46 +22,96 @@ A pure Python micro-framework supporting seamless lazy and concurrent evaluation
     :target: https://www.codacy.com/manual/Othoz/paragraph?utm_source=github.com&amp;utm_medium=referral&amp;utm_content=Othoz/paragraph&amp;utm_campaign=Badge_Grade
 
 
-In essence, the package allows to write *functional* code directly in Python: statements merely specify relationships among *variables* through *operations*.
-Evaluation of any variable given the values of other variables is then de facto:
+Introduction
+''''''''''''
 
-- **lazy**: only operations participating in the determination of the requested value are executed,
-- **concurrent**: operations can be executed by a thread pool of arbitrary size.
+Paragraph adds the *functional programming paradigm* to Python in a minimal and almost seamless fashion. One additional class, ``Variable``, and a
+function decorator, ``op``, is all it takes to turn regular Python code into a *computation graph*, i.e. a computer representation of a system of
+equations:
 
-In addition, relationships among variables can be traversed in both directions, allowing a form of backpropagation of
-information through the computation network that would be cumbersome to implement in an imperative manner.
+>>> import paragraph as pg
+>>> import operator
+>>> x, y = pg.Variable("x"), pg.Variable("y")
+>>> add = pg.op(operator.add())
```

Comment:
Is it correct to call `operator.add` here?

Reply:
Oops, good point!